src/binder/statement/create_table.rs

```diff
@@ -143,7 +148,7 @@ impl Binder {
                     columns,
                     ..
                 } if *is_primary => columns.iter().for_each(|ident| {
-                    pks_name_from_constraints.push(ident.value.clone());
+                    pks_name_from_constraints.push(ident.value.to_lowercase());
```

Comment:
I think we can keep everything case-sensitive for now. I'd prefer not to convert to lowercase.

Reply:
done

Reply:
done